apps/website/src/pages/docs/carousel.mdx

```diff
@@ -14,64 +14,80 @@ import * as AllExamples from 'examples';
+The Carousel component consists of a set of slides, normally displayed one at a time. A navigation section is located below the slides, consisting of ["dots"](#dots-list) and ["previous"/"next"](#navigation) buttons, used for changing slides.
+
+The Carousel component is made up of [customizable subcomponents](#component-api). Here's what the basic usage looks like:
 
-The currently shown slide can also be changed using the left/right arrow keys or by dragging on a touch device.
+```tsx
+<Carousel>
+  <Carousel.Navigation />
+  <Carousel.Slider>
+    <Carousel.Slide>…</Carousel.Slide>
+    <Carousel.Slide>…</Carousel.Slide>
+    <Carousel.Slide>…</Carousel.Slide>
+  </Carousel.Slider>
+</Carousel>
+```
 
 ## Usage
 
-Carousels can be controlled externally by using the properties `activeSlideIndex` and `onSlideChange`.
+### Controlled State
+
+By default, the carousel maintains its own state. But it can also be controlled externally using `activeSlideIndex` and `onSlideChange`.
 
 <LiveExample src='Carousel.controlled.tsx'>
   <AllExamples.CarouselControlledExample client:load />
 </LiveExample>
 
-The subcomponents can be used outside of Carousel. The following example shows how `Carousel.DotsList` can be used without the `Carousel` component. Clicking on the right half of the slide will advance the carousel to the next slide, whereas clicking on the left half will go to the previous slide.
+## Accessibility
 
-<LiveExample src='Carousel.onlyDots.tsx'>
-  <AllExamples.CarouselOnlyDotsExample client:load />
-</LiveExample>
+The `Carousel` component is built with accessibility in mind and tested to work well with different input modalities. It includes support for keyboard users and touch-screen users.
 
-## Props
+The component is loosely based on the [APG Tabs pattern](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/), where the dots are considered to be "tabs" and the slides are considered to be "panels". For this reason, it is important that the dots (navigation section) is present before the slides in DOM order.
 
-Properties table is to be defined.
+When a slide is changed, all inactive slides will become ["inert"](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inert) so that they cannot be interacted with by users of assistive technology.
 
-<PropsTable path={frontmatter.propsPath} />
+Lastly, the carousel does not support auto-rotating slides, because that can lead to many different [accessibility](https://www.boia.org/blog/carousels-and-accessibility-7-tips) and [usability](https://www.nngroup.com/articles/auto-forwarding/) issues. We do _not_ recommend implementing your own auto-rotating carousel unless you have done extensive research and testing with users (including users with disabilities).
 
-## Subcomponents
+## Component API
 
-This component uses a composition approach so it should be used with the provided subcomponents.
+### Carousel
```

Comment:
The "Carousel" heading could use some text, even if it's a single sentence.

Reply:
no, that text is unrelated to the `Carousel` (wrapper) part of the component api.

i was thinking something like:
> `Carousel` is the main component that orchestrates all other sub-components and maintains the overall state.